### DISCUSSION TITLE
pi400-Audio Jack

### DISCUSSION THREAD

**Discussion by BaskaranProdigy** [Pi4J/pi4j#243] [Q&A]
Developing Carrier Board using Pi400
I need to implement Audio Input Connector ,
Could you advice which is  Audio Signal Pin from the 40 Pin on  pi400 PCBA
Share Pi400 Module to Audio Input signal Schematic 
Thanks
K.Baskaran

**Comment by taartspi** [comment]
https://pi4j.com/1.4/pins/rpi-400.html#   I did not find a schematic for the pi 400 board.  But the raspberry web site describes it as a Pi4 using the same 40 pin design meaning no analog to digital conversion is built into the board.  In the Pi4j web pages the Examples section show implementations to connecting to an external A/D chip. And there are other chips available for this purpose that might better suite your goals

**Reply by BaskaranProdigy** [reply]
Thank you for your Brief answer
